core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.ts

```diff
@@ -100,7 +100,9 @@ angular.module('oppia').component('translationOpportunities', {
           },
           controller: 'TranslationModalController'
         }).result.then(function() {
+          document.body.style.cursor = '';
```

Comment:
Can you put this is a new function with a descriptive name actually so its purpose is clear, e.g. resetCursorStyle() and then add a comment in the function explaining that it's needed to reset the cursor after using the copy tool in the modal?

Reply:
Sure, I'll add the statement inside a function.

Reply:
One question though. Should I edit the template as well (since you closed the copy tool issue)?

Reply:
I've addressed the changes and added the statement inside a function and re-used it. The comments should help.